content/api/hub/sample_router.md

```diff
@@ -5,24 +5,24 @@ content-type = "text/html"
 tags = ["python", "http", "routing"]
 
 [extra]
-author = "technosophos"
+author = "fermyon"
 type = "hub_document"
 category = "Sample"
 language = "Python"
 created_at = "2023-12-08T00:00:00Z"
 last_updated = "2023-12-08T00:00:00Z"
 spin_version = ">v2.0"
 summary =  "A sample that shows how to do internal HTTP routing in a Spin Python application"
-url = "https://github.com/technosophos/example-router-py"
+url = "https://github.com/fermyon/spin-python-sdk/tree/main/examples/external-lib-example"
```

Comment:
Is this showing a router or showing how to use an external library?

Reply:
It does both. It uses an external library and how it can be used to route.

Reply:
Hmm, okay.  I am concerned that if someone comes to the repo looking for "that router example they showed" then it's going to be highly non-obvious where to find it, but maybe that's managed through a read-me or something.  And I guess is out of scope for the Hub card.

Reply:
Not sure I follow, It redirects directly to the example.

Reply:
No, sorry, this is out of scope for the Hub card but is musing on someone going to the repo and looking for things (which, realistically, is what many people do).  They are thinking about "router" and they see this:

![image](https://github.com/fermyon/developer/assets/865538/d93abdc9-dbf1-4930-afac-ff9ee0b493be)

But I agree this is a discussion for the SDK repo not for the Hub card.